templates/parachain/zombienet-omni-node.toml

```diff
@@ -0,0 +1,22 @@
+[relaychain]
+default_command = "polkadot"
+chain = "rococo-local"
+
+[[relaychain.nodes]]
+name = "alice"
+validator = true
+ws_port = 9944
+
+[[relaychain.nodes]]
+name = "bob"
+validator = true
+ws_port = 9955
+
+[[parachains]]
+id = 1000
+chain_spec_path = "<path/to/chain_spec.json>"
```

Comment:
Maybe not a bad idea to provide sample chain-spec file within the repo? Could also be used (or refered when talking about `chain-spec-builder` in `parachain/README.md`.

Reply:
Same as https://github.com/paritytech/polkadot-sdk/pull/5954/files#r1819601935